feat(deploy) update release note (#1065)
bonitasoft/bonita-doc: md/project_deploy_in_studio.md

# Project deployment in a Studio

> This page presents an overview of the _Deploy wizard_ capabilities in the Studio

Reviewer: ```suggestion
This page presents an overview of the _Deploy wizard_ capabilities in Bonita Development Suite.
```

Reviewer: or "This page presents an overview of the _Deploy wizard_ capabilities that allow the deployment of a project, or parts of a project, from Bonita Studio to the embedded Tomcat."

## Artifact selection

> After importing or cloning a project in the Studio, you will have the possibility to directly deploy your project in the embedded studio runtime.  

Reviewer: either "Bonita Studio" or "the studio".
No "the Bonita Studio", 
No "the Studio"
No "Bonita studio"
#shrug :]

> After importing or cloning a project in the Studio, you will have the possibility to directly deploy your project in the embedded studio runtime.  

Author: ```suggestion
After importing or cloning a project in the studio, you will have the possibility to directly deploy your project in the embedded studio runtime.  
```

> You can open the _Deploy wizard_ using the contextual menu on the project in the Project Explorer view or using find the action directly in the toolbar bar.  

Reviewer: ```suggestion
You can open the _Deploy wizard_ using the contextual menu of the project in the Project Explorer view or find the action directly in the toolbar.  
```

Reviewer: what toolbar? we still call it the "Coolbar".
Or you need to replace it everywhere by "the main toolbar", as we have contextual toolbars in some editors.

Author: We can't do that. It is a toolbar

In the wizard, you have the possibility to select or deselect an artifact. An artifact can be a process, a page or any deployable element of your project.  

> Processes are categorized by versions and you can opt to only select the latest version ([SemVer](https://semver.org/)). The artifact selection is persisted between to deployment. You may also use the search text to quickly find an artifact in a large project.

Reviewer: ```suggestion
Processes are categorized by versions and you can opt to only select the latest version ([SemVer](https://semver.org/)). The artifact selection is persisted between two deployments. You may also use the search text to quickly find an artifact in a large project.
```
